ansible/roles/cli/tasks/copy_local_openwhisk_cli.yml

```diff
@@ -5,6 +5,20 @@
 - local_action: stat path={{ openwhisk_cli.local.location }}/{{ item }}
   register: binary_path
 
+- name: "ensure Nginx cli directory is writable"
+  file:
+    path: "{{ cli.nginxdir }}"
+    state: directory
+    mode: 0777
+  become: true
+
 - name: "copy the local binaries from a local directory to Nginx directory"
   copy: src={{ openwhisk_cli.local.location }}/{{ item }} dest={{ cli.nginxdir }}/
   when: binary_path.stat.exists
+
+- name: "modify the mode of Nginx cli directory"
```

Comment:
remove this since you are already setting permission to `0777` before on the same directory

Reply:
Is it good to keep the folder 0777, which means anyone can do anything? We set it to 0777 in order to upload the binaries. After it is done, do we still keep the 0777?